suites/nautilus/dashboard/sanity_containerized_ceph_dashboard.yaml

```diff
@@ -0,0 +1,103 @@
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+        name: containerized ceph with dashboard
+        polarion-id: CEPH-83573534
+        module: test_ansible.py
+        config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: repository
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              containerized_deployment: true
+              copy_admin_key: true
+              ceph_docker_registry: "registry.redhat.io"
+              ceph_docker_registry_auth: true
+              ceph_docker_registry_username: <provide_username>
+              ceph_docker_registry_password: <provide_password>
+              dashboard_enabled: True
+              dashboard_admin_user: admin
+              dashboard_admin_password: p@ssw0rd
+              grafana_admin_user: admin
+              grafana_admin_password: p@ssw0rd
```

Comment:
Added default username and password so that once https://github.com/red-hat-storage/cephci/issues/189 this issue got fixed we can also add this to Jenkins,Please let me know Is it good to merge now?

Reply:
Sunil Angadi, Can you take up this issue #189

Reply:
@rakeshgm @XDsubhash I think subhash has some workaround, Sure I will take it up.

Reply:
OK. Csn you assign the issue to yourself.

Reply:
yes assigned on my name